storage/src/journal/fixed.rs

```diff
@@ -699,10 +706,6 @@ impl<E: Storage + Metrics, A: Codec<Cfg = ()> + FixedSize> Journal<E, A> {
 
     /// Syncs and closes all open sections.
     pub async fn close(self) -> Result<(), Error> {
-        for (i, blob) in self.blobs.into_iter() {
-            blob.sync().await?;
-            debug!(blob = i, "synced blob");
-        }
```

Comment:
Not sure if we still want to keep this and just add a comment that it's a no-op?